src/chapters/errors-and-debugging/asking-good-questions.rst

```diff
@@ -0,0 +1,42 @@
+Asking Good Questions
+=====================
+
+If employing the strategies outlined in this chapter still don't help you find a bug in your code, you always have other programmers to reach out to.
+
+Whether you are asking a question of your teaching assistant, another student, or a stranger in an online forum, you should first be able to answer the 3 questions discussed in this section.
+
+Not only will these questions help others assist you more directly, but they may just lead you to the answer yourself! 
+
+What is the problem with your code?
+-----------------------------------
+
+You should be able to state in general terms what the error that you are experiencing is. 
```

Comment:
Grammar: "You should be able to state in general terms the error you are experiencing."